.github/ISSUE_TEMPLATE/frontend_report.yml

```diff
@@ -0,0 +1,75 @@
+name: "🌐 Frontend (Issue / Optimization / Documentation)"
+description: Report an issue specifically related to the website frontend. For other inquiries, please use the Discussions section.
+title: "[Frontend]: "
+labels: "Status: Unconfirmed"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT:** Failure to comply with the following guidelines may result in immediate closure.
+        - Prior to submitting, kindly search the closed issues to check if the problem you are reporting has already been addressed and resolved. If you come across a closed issue that pertains to your problem, please leave a comment on that issue instead of creating a new one.
+        - If the problem is related to a bug in the website frontend, kindly check for browser compatibility and ensure the issue occurs in multiple browsers before submitting.
+        - For suggestions, questions, or feature requests, please use the [Discussions section.](https://github.com/community-scripts/ProxmoxVE/discussions)
+  
+  - type: input
+    id: guidelines
+    attributes:
+      label: Please verify that you have read and understood the guidelines.
+      placeholder: 'yes'
+    validations:
+      required: true
+
+  - type: dropdown
+    id: issue_type
+    validations:
+      required: true
+    attributes:
+      label: What type of issue is this?
```

Comment:
Why restrict the user like this? It could also be something else

Reply:
For Example? 😄 

i can add "Other", but somehow I think the 3 options are already good. But @BramSuurdje can also say something about that

My intention behind it:
I think free text would create a lot of “garbage” here, or it would be misused to directly insert large error parts as text.